### PR TITLE
[CEDS-3314]: Migrate from play-ui

### PR DESCRIPTION
**Pull request by don-mac**
Continued migration away from play-ui.

Added a copy of @errorInLine component to the project to decouple its use from the library.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -38,7 +38,6 @@ val compileDependencies = Seq(
   "uk.gov.hmrc"               %% "bootstrap-frontend-play-28" % "5.3.0",
   "uk.gov.hmrc"               %% "govuk-template"             % "5.66.0-play-27",
```
Comment (udinobi) [inline build.sbt:39]: `5.69.0-play-28`

File: build.sbt [hunk 2]
```diff
@@ -38,7 +38,6 @@ val compileDependencies = Seq(
...
   "uk.gov.hmrc"               %% "play-frontend-hmrc"         % "0.68.0-play-28",
```
Comment (udinobi) [inline build.sbt:40]: `0.94.0-play-28`

**Comment by udinobi** [comment]
Not in the ticket of course, but I would update the other deps as well. But it's just a suggestion.

**Review by udinobi — APPROVED** [review]

**Review by udinobi — APPROVED** [review]

**Comment by don-mac** [comment]
This PR has been superceded by 
 PR https://github.com/hmrc/cds-file-upload-frontend/pull/163
 
 There were unverified commits contributing to a blocked merge status and it was cleaner to start from a new branch and PR

**Pull request closed without merging**